src/v2i-hub/TelematicBridgePlugin/src/TelematicBridgeMsgWorker.h

```diff
@@ -238,7 +240,9 @@ namespace TelematicBridge
             }
             else
             {
-                json["payload"] = StringToJson(cJSON_Print(msg->payload));
+                // Render a cJSON entity to text for transfer/storage. Free the char* when finished.
+                auto payloadPtr = std::make_unique<char*>(cJSON_Print(msg->payload));
```

Comment:
Apparently `std::make_unique` is sort of a wrapper for `new`  (https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique) and actually creates a new object from the passed in parameter. This is likely why we still see the memory leak with this in. Apparently this is the more appropriate way to create a unique pointer from the returned char*
```cpp
 std::unique_ptr<char, decltype(std::free) *> t_copy { cJSON_Print(msg->payload), std::free };
```
Could you try this and see if it works?